alphagov/pay-selfservice
# PP-12735 Change RESTClientError import to pay-js-commons class

## WHAT
Updating the post-configure.controller to use the RESTClientError from pay-js-commons/axios-client so that correct error message is shown to users as opposed to a 500 internal server error message.

## HOW 
_Steps to test or reproduce:_

1. Sign in to an existing user account in the admin tool
2. Follow the steps to set up a new authenticator app until you reach the page where you have to enter the code from the app.
3. Deliberately enter an incorrect code.
4. You should now see the following error message:
> The security code you entered is not correct, try entering it again or wait for your authenticator app to give you a new code
<img width="1292" alt="Screenshot 2024-07-11 at 17 02 15" src="https://github.com/alphagov/pay-selfservice/assets/92534363/4bcd7b94-3929-4c94-abce-f005e03270bb">
<img width="1162" alt="Screenshot 2024-07-11 at 17 02 26" src="https://github.com/alphagov/pay-selfservice/assets/92534363/0ace1b6b-3ed8-4c25-a926-cc0194f43ade">


_Note: There are other instances of the class that was causing this bug which are being addressed in another ticket_ [here](https://payments-platform.atlassian.net/browse/PP-12868).
